### ISSUE TITLE
Plot cuts off validation points

### ISSUE THREAD

**Issue by kickbox** [rstudio/keras#54]
My validation points are not shown in the plots. This may be because the y-axis starts from a higher value.

```
> history
Trained on 109,892 samples, validated on 12,211 samples (batch_size=128, epochs=25)
Final epoch (plot to see history):
     acc: 0.8822
    loss: 0.3102
val_loss: 1.389
 val_acc: 0.6713
```
I tried using `ylim `attribute but it doesnt seem to work

`> plot(history, ylim=c(0,.9))`

![image](https://user-images.githubusercontent.com/7101002/27622465-e8b4c7fc-5bf4-11e7-9ffb-4fc4a1c5687d.png)

**Comment by Ax3man** [comment]
The y-axis is only scaled by the training data and not the validation data. The plot function does not pass on additional `...` parameters to the plot function.

You can see the source by typing: `keras:::plot.keras_training_history`.

**Comment by jjallaire** [comment]
I am now passing `...` on to the `plot` function (nice catch!).

Can either of you suggest a way to get better automatic scaling here (i.e. don't require the `ylim` tweak for most cases)

**Comment by Ax3man** [comment]
Yes, I can prepare a PR somewhere in the next week, if you'd like. 

On a related note, would you also be interested in a ggplot method?

**Comment by jjallaire** [comment]
Yes on both counts!

I'm in general interested in any we can provide additional visualizations
to help with training/evaluation. As far as I'm concerned it would be fine
if these were all based on ggplot2.

**Comment by jjallaire** [comment]
We could do `method = c("auto", "ggplot2", "base")` and then select ggplot2 when it's installed (that way we wouldn't carry a ggplot2 Imports dependency).

I am not enough of a DL practitioner to say what generic visualization infrastructure is required/beneficial (I took the first one from a book that provided the comparable matplotlib code).

**Comment by Ax3man** [comment]
That makes sense, will do. I should have a PR relatively soon then.

I'm starting playing around with these nets last week so very little expertise here, I'm afraid. But there's at least some options, looking at what h2o supplies in their browser interface.

**Comment by jjallaire** [comment]
Okay, thanks. Much appreciated!
